qcodes/instrument_drivers/Keysight/keysightb1500/KeysightB1500_base.py

```diff
@@ -64,6 +63,17 @@ def __init__(self, name, address, **kwargs):
 
         self.connect_message()
 
+    def write(self, cmd):
```

Comment:
anything we need to do for "ask" calls?

Reply:
I think no. 1) Will only be dangerous only if instrument throws sensible value but store some error in the buffer. For all the measurements we have written, we use 'XE' command. Its output already contains which data is normal "N" and which one has problems by storing strings such as "G", "V" ... in compliance setting. Hence from the measurement side, we are safe. From the setting point of view, we are already checking the error while setting it. So maybe we are ok if we do not check it again while trying to read the settings.

Reply:
great!